ci.yml

```diff
@@ -0,0 +1,12 @@
+script:
+        - cd build
+        - echo $PATH
+        - export HOME=$(dirname "$PWD")
+        - oc login -u kubeadmin -p BdTY8-Vra64-gD3cI-dAPS6 --insecure-skip-tls-verify=true https://api.crc.testing:6443
```

Comment:
Technically this password is actually config owned by the CI server, right? Can we have it passed in through the provided environment (like HOME) instead of hard coding into every ci.yml that wants to use the system?

Reply:
the password is included in the file ~/.crc/cache/<version>/kube.....

Reply:
Just to help clarify; the code in this ci.yml is running from a ci pod in openshift that does not have ready access to that file. The suggestion is for the for the ci server itself to provide it via the environment so it can be made use of in ci scripts without hard-coding it in each.

To achieve this we should document in the README how to generate an openshift secret from the file Matthias mentions (from the context of the machine running CRC), and the ci server should consume this secret into it's environment and expose it in the environment in passes down to the ci scripts.

Not suggesting this needs to happen before this PR merges, I'd just prefer to see it happen before the 2nd or 3rd instance of this password gets copy/pasted around.

PS. This password is, as far as we can tell, actually shared among all CRC instances from the same build and is in no way secret, so it's appearance in this repo should not be a major concern; though it's not a practice we should like to see in general.

Reply:
I see a kubeconfig at ~/.crc/machines/crc/kubeconfig, I'm not certain if this is a permanent one associated with crc? If so, that can be used to create the secret used by deploy-on-k8s.yml and then we wouldn't need that login command in CI.yml anymore. I'll check into it for a future PR